applications/plugins/image/CMakeLists.txt

```diff
@@ -3,6 +3,8 @@ project(image VERSION 0.1)
 
 find_package(SofaBase REQUIRED)
 find_package(SofaGeneral REQUIRED)
+find_package(OpenGL QUIET)
```

Comment:
Don't forget to handle this new dependency in imageConfig.cmake.in
Also, you should use `sofa_find_package`, particularly when it's a soft dependencies. Otherwise, you won't have `IMAGE_HAVE_OPENGL` in imageConfig.cmake.in